ale_linters/ocaml/merlin.vim

```diff
@@ -0,0 +1,15 @@
+" Author: Andrey Popp -- @andreypopp
+" Description: Report errors in OCaml code with Merlin
+
+function! ale_linters#ocaml#merlin#Handle(buffer, lines)
+  let l:errors = merlin#ErrorLocList()
+  return l:errors
+endfunction
+
+call ale#linter#Define('ocaml', {
+\   'name': 'merlin',
+\   'executable': 'ocamlmerlin',
+\   'command': 'true',
```

Comment:
So in this case we read an error list directly from merlin (which keeps a connection to merlin server and uses it to fetch errors). This is why we use `true` command. Hacky... but works.